frontend/src/components/account-mgmt/ManageUserDialog.vue

```diff
@@ -448,11 +437,8 @@ export default {
      */
     async doesUserExist(firstName, lastName, email) {
       try {
-        // TODO (jgstorey) Make this a service function
-        const res = await ApiService.apiAxios.get(
-          `${ApiRoutes.ORGANIZATIONS}${ApiRoutes.ORGANIZATIONS_USERS.replace(':organizationId', this.userInfo.organizationId)}?firstName=${firstName}&lastName=${lastName}&email=${email}`,
-        )
-        if (Array.isArray(res.data) && res.data.length >= 1) {
+        const res = await OrganizationService.getOrganizationUsers(this.userInfo.organizationId, firstName, lastName, email)
+        if (Array.isArray(res) && res.length >= 1) {
           return true
```

Comment:
it's not your change, but it is simpler to just have "return (Array.isArray(res) && res.length >= 1)" instead of return true and return false

Reply:
this.errorMessages = [] is probably not needed since it will be auto reset in checkBCeIDExists()

Reply:
also, I think there is a bug when user tries to create a duplicate user BCeID. it is an edge case so we probably can fix it later.

Reply:
also, we're sending an API request to checkBCeIDExists() for every time the user updates the bceid name, then we send another API request to getOrganizationUsers in doesUserExist() when the user click "Add" --> We can reduce these number of calls by saving all organization users in a variable then use it in checkBCeIDExists. (probably a tech spike ticket?)

Reply:
this bug is fixed in my commit.